### ISSUE TITLE
Write a test Jenkinsfile

### ISSUE THREAD

**Issue by jaseemabid** [fabric8io/fabric8-build-team#29]
A rudimentary test Jenkinsfile file

- [ ] This needs to be simple and well documented
- [ ] Covers a good share of features
- [ ] Small enough to finish in the next sprint

**Comment by sthaha** [comment]
@jaseemabid could you please shed some light into what we are trying to achieve with this Jenkinsfile?

**Comment by jaseemabid** [comment]
@sthaha The current test setup (godog-jenkins) is too messy and unreliable, so I was hoping to write a simple straight forward Jenkinsfile which runs and deploys a quick start app for quick end to end testing. 

- [ ] Make it configurable such that we can run it with specific versions of fpl and fmp. 
- [ ] Curl the final deployed application and make sure its up
- [ ] Test re-release of the app with a small patch to the app

PS: I filed this bug before I digged into godog-jenkins and I think its quite useless. I'l rather throw it away now and mark this a duplicate of https://github.com/fabric8io/fabric8-build-team/issues/45
